src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -93,6 +93,17 @@ def _reset(self):
 
     #############################################################################
 
+    def setToken(self, token, valid=0):
```

Comment:
Does it make sense to have `valid=0` as an optional argument? If the default is 0 I guess it always needs to be set:
```suggestion
    def setToken(self, token, valid):
```